hw/ip/prim/prim.core

```diff
@@ -45,6 +47,8 @@ filesets:
       - rtl/prim_subreg.sv
       - rtl/prim_subreg_ext.sv
       - rtl/prim_intr_hw.sv
+      - rtl/prim_rom_adv.sv
+      - rtl/prim_ram_1p_adv.sv
```

Comment:
Please revert the change you made to `prim_ram_1p_adv.sv` and leave it a separate file.

Reply:
Sorry shouldn't have force pushed in between changes. I haven't forgotten to do that, but wanted to get CI going in between.

Reply:
Ah sorry, it was me being too eager then.